airflow/__init__.py

```diff
@@ -23,39 +23,39 @@
 implement their own login mechanisms by providing an `airflow_login` module
 in their PYTHONPATH. airflow_login should be based off the
 `airflow.www.login`
-"""
 
-from typing import Optional, Callable
-from airflow import version
-from airflow.utils.log.logging_mixin import LoggingMixin
+isort:skip_file
```

Comment:
Does this now show up in the API section of the rendered docs?

Is there a reason we are skipping this file but making changes to it? Did perhaps we need a blank line first?

```suggestion

isort:skip_file
```

Reply:
There're a lot implied dependencies inside multiple imports in this file. I tried a couple times try to reorder a couple but it keeps getting me more errors so I decided to skip it with the maximum changes I tried that won't break it so I don't have to add expection comment in multiple lines. And I'll for sure add that blank line :P

Reply:
And sry I'm missing the docs part. Do you mind help pointing me to this rendered docs you referred to please?

Reply:
BTW. I just merged the pylint changes few days ago - so rebasing this one and re-running the isort should be good to go @KevinYang21

Reply:
👍 Thank you